runway/cfngin/hooks/aws_lambda.py

```diff
@@ -2,19 +2,29 @@
 import hashlib
 import logging
 import os
-import os.path
 import stat
+import subprocess
+import sys
 from io import BytesIO as StringIO
+from shutil import copyfile
 from zipfile import ZIP_DEFLATED, ZipFile
 
 import botocore
+import docker as Docker  # noqa: N812
```

Comment:
Seem unnecessary to do the docker -> Docker import rename (would prefer to get rid of the flake8 exception)